src/include/miopen/sqlite_db.hpp

```diff
@@ -55,9 +56,14 @@ class path;
 } // namespace boost
 
 namespace miopen {
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_DISABLE_SQL_WAL)
 
-constexpr bool InMemDb                = MIOPEN_EMBED_DB;
+constexpr bool InMemDb = MIOPEN_EMBED_DB;
+#if MIOPEN_ENABLE_SQLITE_BACKOFF
+const auto MIOPEN_SQL_BUSY_TIMEOUT_MS = 10;
```

Comment:
When `MIOPEN_ENABLE_SQLITE_BACKOFF` is true. this should pass the timeout to `LazyExponentialBackoff` constructor instead of to sqlite.

Reply:
With the self implemented backoff, SQLite still requires a timeout to wait until it returns SQLITE_BUSY. When there is no backoff that timeout is 60 seconds, when there is backoff that timeout is 10ms so that the handler would be invoked sooner.